### PR TITLE
Update user groups for demo editor

### PR DESCRIPTION
**Pull request by Evidentsinger14**
Aberdeener - group.mod
ImDarkLaw + group.mod
Leotomas + group.mod
Ev1dent + group.mod

### CODE REVIEW

**Review by powercasgamer — APPROVED** [review]

File: src/data/editor-demo.json [hunk 1]
```diff
@@ -436,42 +436,32 @@
     },
     {
       "type": "user",
-      "id": "ea3ff55d-f4bc-4fab-a6ae-d809d0849f95",
-      "displayName": "powercas_gamer",
+      "id": "dea5c5f3-8686-48ec-a7e7-ec19ed805bed",
+      "displayName": "Leotomas",
```
Comment (LeotomasMC) [inline src/data/editor-demo.json:440]: i actually have changed my username https://namemc.com/profile/BNTFryingPan.1

**Review by LeotomasMC — CHANGES_REQUESTED** [review]

File: src/data/editor-demo.json (hunk 1 above)
Comment (LeotomasMC) [reply src/data/editor-demo.json:440]: weird git diff lmao

Comment (Evidentsinger14) [reply src/data/editor-demo.json]: Fixed, ty

**Comment by Turbotailz** [comment]
TYVM

**Pull request merged**